templates/commands/goldentest/verify.go

```diff
@@ -163,6 +165,17 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) error {
 			}
 		}
 
+		stdOutDiff, err := getStdoutDiff(goldenStdoutFile, tempStdoutFile, dmp)
```

Comment:
This function is getting awfully long (135 lines), could you factor out one or more of the `for` loops into separate function(s)? This could be a followup PR.

Reply:
Will do it in another PR.